build/ftest-base/pom.xml

```diff
@@ -32,7 +32,7 @@
     <arquillian.contaner.maxTestClassesBeforeRestart>50</arquillian.contaner.maxTestClassesBeforeRestart>
 
     <!-- Container Selection -->
-    <arquillian.launch.jbossas71>false</arquillian.launch.jbossas71>
+    <arquillian.launch.wildfly261>true</arquillian.launch.wildfly261>
```

Comment:
how about name it `...wildfly` without the version? I think it will be easier to maintain and evolve in the future.

The version of WildFly will be determined by `version.wildfly` property in pom.xml.

Reply:
Done. Without the version number, there will always be only one WildFly server testable ;-).

Reply:
> 

True. I think we can have more frequent releases to support latest WildFly, or other servers. Users can choose different versions of arquillian-warp for different versions of WildFly.